quarkus/server/src/main/resources/default-local-cache.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2019 Red Hat, Inc. and/or its affiliates
+  ~ and other contributors as indicated by the @author tags.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<infinispan
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="urn:infinispan:config:9.4 http://www.infinispan.org/schemas/infinispan-config-9.4.xsd"
+        xmlns="urn:infinispan:config:9.4">
+
+    <cache-container name="keycloak">
+        <local-cache name="default">
+            <transaction transaction-manager-lookup="org.infinispan.transaction.lookup.JBossStandaloneJTAManagerLookup"/>
+        </local-cache>
+        <local-cache name="realms">
+            <memory>
+                <object size="10000"/>
+            </memory>
+        </local-cache>
+        <local-cache name="users">
+            <memory>
+                <object size="10000"/>
+            </memory>
+        </local-cache>
+        <local-cache name="sessions"/>
```

Comment:
If there is a way to avoid 3 additional files in the workspace with those configurations duplicated, it could be nice. We already have it in few more places, which means that adding/removing/updating any infinispan cache is more and more complicated with more and more possibilities that some place in the workspace will be forgotten.

I guess it would be a non-trivial effort to do it, so I am fine with accepting this PR even with this change. But it will be a nice "bonus" if it is possible to avoid it :)

Reply:
The config stuff here is a bit temporary and will change once we drive config through Keycloak config file. I agree we shouldn't have duplicated/multiple versions of this file in the repo.

Reply:
I see your point. This specific config (and the other default one) is meant for running the executable JAR directly or when running in dev mode.

I'm not sure too how to avoid this duplication.

Reply:
Ok, Thanks for the explanation.